### PR TITLE
James pr xamarin sample/ra

### LINKED ISSUE #21443: Update to 3.1  Create backend services for native mobile apps
Internal notes:

* https://review.docs.microsoft.com/en-us/help/contribute/code-in-docs?branch=master#snippet-syntax-reference
* https://review.docs.microsoft.com/en-us/help/onboard/admin/cross-repository-reference?branch=master

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bdbdf1ed-c52a-3be8-1f41-aecbdab0883b
* Version Independent ID: d06996cc-14aa-96e2-fd62-9d5bbc059107
* Content: [Create backend services for native mobile apps with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/mobile/native-mobile-backend?view=aspnetcore-5.0)
* Content Source: [aspnetcore/mobile/native-mobile-backend.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/mobile/native-mobile-backend.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-mobile**
* GitHub Login: @ardalis
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #21443
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/mobile/native-mobile-backend?view=aspnetcore-6.0&branch=pr-en-us-21489)

### CODE REVIEW

File: aspnetcore/mobile/native-mobile-backend/sample/TodoAPI/src/TodoAPI/TodoAPI.csproj [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (Rick-Anderson) [inline aspnetcore/mobile/native-mobile-backend/sample/TodoAPI/src/TodoAPI/TodoAPI.csproj:4]: @scottaddie  as you can see there is a project file. But the snipets 500 check returns the error

> Error: 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code.

**Comment by Rick-Anderson** [comment]
@jamesmontemagno  https://docs.microsoft.com/en-us/aspnet/core/mobile/native-mobile-backend is live now.

**Comment by jamesmontemagno** [comment]
Whoop whoop!!! YAY! Thanks Rick for the hard work here! <3

**Comment by jamesmontemagno** [comment]
@Rick-Anderson  I think the only other thing I had in my PR was an update for the File->New project in VS (maybe we should add VSM and also cli?)

**Comment by Rick-Anderson** [comment]
I can do that per #21584
 What's VSM? I can add VSC or CLI. Given VSC and cli are pretty much the same, we only add one.

**Comment by jamesmontemagno** [comment]
Visual Studio for Mac :)

**Pull request merged**